themes/bootstrap5/templates/_ui/components/hide-offcanvas-button.phtml

```diff
@@ -1,7 +1,11 @@
 <?php
     $side = $this->config()->offcanvasSide();
+    $buttonAttrs = [
+        'class' => 'close-offcanvas btn btn-link ' . $this->_themeClass,
+        'data-bs-toggle' => 'vufind-offcanvas',
```

Comment:
I notice that `data-toggle` is changing to `data-bs-toggle` here. Is this intentional, or is the automatic bootstrap3/bootstrap5 conversion doing something unexpected? (Just asking because the templates are now inconsistent across the two themes, but they previously were not, and yet the automated checks are passing).

Reply:
hmm, good catch as i used to update-bootstrap5 command with these, ill check

Reply:
seems like it was data-toggle="collapse" in 3.4.1 and data-bs-toggle="collapse"  in 5. I think the update script thought that this data was bootstrap specific. Will adjust

Reply:
Sounds good. And if the update script won't let you for some reason, you might want to talk to @EreMaijala about it.

Reply:
Ah, yes, there's a conflict in bootstrap3 where data-toggle is used by both VuFind and Bootstrap. Fortunately it won't be a problem that much longer. :)